spec/models/file_set_spec.rb

```diff
@@ -221,34 +221,52 @@ class AltFile < ActiveFedora::Base
 
   describe 'noid integration', :clean_repo do
     let(:service) { instance_double(ActiveFedora::Noid::Service, mint: noid) }
+    let(:noid) { 'wd3763094' }
+    let(:default) { Hyrax.config.enable_noids? }
```

Comment:
Maybe use `let!` here to ensure this happens before anything in specs (as it in a before block)?